src/node/history.h

```diff
@@ -550,10 +540,7 @@ namespace ccf
                 }
                 case kv::Consensus::SignatureDisposition::CAN_SIGN:
                 {
-                  if (
-                    self->store.committable_gap() > 0 &&
-                    time > time_of_last_signature &&
-                    (time - time_of_last_signature) > sig_ms_interval)
```

Comment:
The minimal change here would be to change this `>` to a `>=`. With a strict inequality, we came in here exactly when we were supposed to (as far as our timers were concerned), but didn't trigger a signature this time! So we said "sign every 100ms", after 100ms we came here with a commit gap and said "the time gap is 100ms, but I need > 100ms, I'll come back next time". And next time is 100ms, so we double the effective timer.

After making that, I stared at the use of these timers and decided it's all unnecessary. We want this to run at a flat "every 100ms". If we could update it to _delay_ when a different path created a signature, we would, but we can't. So there's no reason to correct or update the inter-callback delay - we don't get accidentally woken up early, and if we _did_, we might as well try to sign _now_, and come back in 100ms. The previous logic was, afaict, trying to say that if we'd said 100ms, and somehow got executed after 20ms, we should only wait 80ms to check again? Sounds wrong to me, an unnecessary.